libs/ui/layout/src/lib/block-list/block-list.component.ts

```diff
@@ -71,14 +78,50 @@ export class BlockListComponent implements AfterViewInit, Paginable {
     })
   }
 
-  // pageIndex is 1-based
+  protected refreshSubComponentsVisibility = () => {
+    this.blocks.forEach((subComponent, index) => {
+      const element = subComponent.nativeElement
+      element.style.display =
```

Comment:
Not sure if this is a good practice to override css style  in js from the great parent. Can't this be done in Angular, with a ngif directive instead?